File: DSA/Arpit_chawla_590015121/day2_q2.c
#include <stdio.h>

int main() {
    int n, i;

    // Ask the user for size of the array
    printf("Enter the number of elements in the array: ");
    scanf("%d", &n);

    int arr[n];

    // Input array elements
    printf("Enter %d elements:\n", n);
    for (i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    // Print the first and last element
    printf("First element: %d\n", arr[0]);
    printf("Last element: %d\n", arr[n - 1]);

    return 0;
}

File: DSA/Muskan_590015043/Day2_Q2.py
#INPUT ARRAY
size = int(input("Enter the number of elements in the array: "))
nums = [] #Initialise empty list
for i in range(size):
    nums.append(int(input()))
first = nums[0]
last = nums[len(nums)-1]
print(f"First: {first}, Last: {last}")